static/scripts/rewards/render-transaction/read-claim-data-from-url.ts

```diff
@@ -31,7 +31,11 @@ export async function readClaimDataFromUrl(app: AppState) {
 
   app.claims = decodeClaimData(base64encodedTxData).flat();
   app.claimTxs = await getClaimedTxs(app);
-  app.provider = await useFastestRpc(app);
+  try {
+    app.provider = await useFastestRpc(app);
+  } catch (e) {
+    toaster.create("error", `${e}`);
+  }
   if (ethereum) {
```

Comment:
I'm having second thoughts about myself having had renamed window.ethereum to ethereum. Could be shadowed easily 😰